botocore/model.py

```diff
@@ -321,6 +321,20 @@ def signature_version(self, value):
         self._signature_version = value
 
 
+def get_streaming_body(shape):
```

Comment:
Not a fan of having this be a module function.  This seems like a method that should live on a shape class.